INSTALLATION.md

```
@@ -13,44 +71,535 @@ You must have basic competence and experience in the following technologies to b
+   ```
+   http://localhost:4000/
+   ```
+
+   - If you navigate to the endpoint you and see a `JSON` response like this it means talawa-api is running successfully:
+
+     ```
+     {"talawa-version":"v1","status":"healthy"}
+     ```
+
+2. The url for accessing the GraphQL Playground is
+   ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)